proto/cusf/mainchain/v1/wallet.proto

```diff
@@ -100,3 +107,47 @@ message GenerateBlocksRequest {
 
 message GenerateBlocksResponse {
 }
+
+message GetBalanceRequest {
+}
+message GetBalanceResponse {
+  uint64 confirmed_satoshi = 1;
+  uint64 pending_satoshi = 2;
+}
+
+message TransactionConfirmation {
```

Comment:
This should include the block hash / parent block hash, seems like it would be important for re-orgs